packages/@react-spectrum/card/src/GridLayout.tsx

```diff
@@ -45,30 +40,46 @@ export interface GridLayoutOptions extends BaseLayoutOptions {
    * @default Infinity
    */
   maxColumns?: number,
+  // TODO: Should the below apply as horizontal item padding if cardOrientation is horizontal or should there be two separate options?
```

Comment:
I think it's fine just to have just one? but maybe update the description to say
`The padding along the items main axis.`
Using terminology from Flex https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox
If they are independently adjustable though... then you'd need both